CONTRIBUTING.md

```diff
@@ -0,0 +1,39 @@
+# Contributing to Aloxide
+
+1. [Getting Involved](#getting-involved)
+2. [How To Develop Features](#how-to-develop)
+3. [How To Report Bugs](#how-to-report-bugs)
+
+## Getting Involved
+
+We're always looking for help [implementing features](#how-to-develop), [identifying bugs](#how-to-report-bugs), writing test cases, and improving documentation. If you find any issues, please don't hesitate to discuss with us, any ideas are helps for us.
+
+There're some related guides for you to check before contributing to our project:
+- [High-level Architecture](https://github.com/lecle/aloxide/wiki/High-level-Architecture)
+- [Commit convention]()
+
+## How To Develop features
+
+### Changing existing/planning features
+
+Make sure you already take a deep look at our [High-level Architecture](https://github.com/lecle/aloxide/wiki/High-level-Architecture). We, as an on-going project, welcome anyone help us improve our code, ideas or design.
+
+### Adding new ones
+
+It would be great to hear new ideas, features contribute to Aloxide, if you have any, don't hesitate to let us know about that.
+
+## How To Report Bugs
+
+### Make sure it is an Aloxide bug
+
+Most bugs reported maybe actually bugs in user code, not in Aloxide code. Keep in mind that just because your code throws an error inside of Aloxide, this does *not* mean the bug is a Aloxide bug.
```

Comment:
It's likely to be true. However, users may want to confirm whether it's a bug or not. Let's guide users to post questions to our groups, https://groups.google.com/g/aloxide, if they're not sure where bug is located or just have some ideas to improve Aloxide.

Reply:
@flowersinthesand Please check if this is okay:

> Any contributions are precious to us, if you have any troubles or just have some ideas to improve Aloxide, please don't hesitate to send us your concerns at our groups https://groups.google.com/g/aloxide, we're happy and willing to help checking at anytime when we're available.

Reply:
LGTM